explorer/ui/app/components/details/TabList.tsx

```diff
@@ -12,13 +12,17 @@ export const TabList = ({ xMsgDetails }) => {
       <div className="flex mt-5 pb-2 border-b-[1px] border-subtle border-solid">
         <p className="w-[150px] sm:w-48 text-sm">Timestamp</p>
         <p className="flex-1">
-          <span className="text-default">
-            {dateFormatter(new Date(xMsgDetails?.block.timestamp))}{' '}
-          </span>
-          (
-          {xMsgDetails?.receipt?.timestamp &&
-            dateFormatterXMsgPage(new Date(xMsgDetails?.receipt?.timestamp))}
-          )
+          {xMsgDetails?.receipt ? (
+          <>
+            <span className="text-default">
+              {dateFormatter(new Date(xMsgDetails?.receipt?.timestamp))}{' '}
+            </span>
+            (
+            {xMsgDetails?.receipt?.timestamp &&
+              dateFormatterXMsgPage(new Date(xMsgDetails?.receipt?.timestamp))}
+            )
+          </>
+          ) : '--'}
```

Comment:
If receipt doesn't exist don't display anything.

Reply:
We'll, we do display `--` : could say something like "No Receipt", or actually nothing. Ask Fab

Reply:
This is what we do for tx_hash already for receipts. I copied that behavior. Happy to change it if required.